src/main/java/org/gridsuite/filter/criteriafilter/VscConverterStationFilter.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.filter.criteriafilter;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.filter.utils.EquipmentType;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+@AllArgsConstructor
```

Comment:
It's technically a `@NoArgsConstructor` as there is no field.

Reply:
optional

Reply:
Done